recipes/plaster/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "plaster" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8351c7c7efdf33084c1de88dd0f422cbe7342534537b553c49b857b12d98c8c3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
As we discussed, it actually uses `pkg_resources` (part of `setuptools`) so we will need to bring it back.